packages/registration_delivery/pubspec.lock

```diff
@@ -427,19 +458,18 @@ packages:
   digit_data_model:
     dependency: "direct main"
     description:
-      name: digit_data_model
-      sha256: e1ea907e9cb7d521341325ac1b1b98fad8bd5e5ca1bab7f7c18ed02ccd2f2c2b
-      url: "https://pub.dev"
-    source: hosted
-    version: "1.0.3"
+      path: "../digit_data_model"
+      relative: true
+    source: path
+    version: "1.0.3-dev.3"
   digit_scanner:
     dependency: "direct main"
     description:
       name: digit_scanner
-      sha256: "1f1fe3ca8f038a3c93cd52491cf72c0b0e45c88b14580243d47eac59672e67ea"
+      sha256: bc172291df5ffe720e3d471fbc5c542086b8efecbdb6e25ee737381cc47c8e05
       url: "https://pub.dev"
     source: hosted
-    version: "1.0.3"
+    version: "1.0.3-dev.1"
```

Comment:
This was moved to stable please do pub get again.

Reply:
> This was moved to stable please do pub get again.

okayy